backend/ed_platform/discourse.py

```diff
@@ -16,8 +16,9 @@ def __init__(self, app):
         self.key = app.config['DISCOURSE']['key']
```

Comment:
This is a breaking change.  I know we had to do this to get it working for you locally, but this should not be a part of the pull request.